transactions/add-keys/2023/nov-16/arguments_service_account.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "type": "Int",
+        "value": "17"
+    },
+    {
+        "type": "String",
+        "value": "f846b840ee4aec89e6d83fe563ee93de01374961fcf117b7e746c78b3097f54b630a053fd4ee70a4bde974ce12d71deb444c5c23c224ad92f05a6f72b54742fd9700a3b7020181fa"
```

Comment:
this encodes the old key, `ee4aec89e6d83fe563ee93de01374961fcf117b7e746c78b3097f54b630a053fd4ee70a4bde974ce12d71deb444c5c23c224ad92f05a6f72b54742fd9700a3b7`

from the readme, it sounds like you should be encoding `475aac6dc9e0a045323f8e24f0aa8ebbec9dfedc34a5cc47bbb206f8d639f03812231731a9608edab6582056ab0a3281350666330daca384c7dff777cc6823be`